x-pack/plugins/security_solution/public/timelines/components/timeline/unified_components/data_table/index.tsx

```diff
@@ -123,24 +126,8 @@ export const TimelineDataTableComponent: React.FC<DataTableProps> = memo(
 
     const showTimeCol = useMemo(() => !!dataView && !!dataView.timeFieldName, [dataView]);
 
-    const tableSettings = useMemo(() => {
-      const columnSettings = columns.reduce((acc, item) => {
-        if (item.initialWidth) {
-          acc[item.id] = { width: item.initialWidth };
-        }
-        return acc;
-      }, {} as Record<string, UnifiedDataTableSettingsColumn>);
-
-      return {
-        columns: columnSettings,
-      };
-    }, [columns]);
-
-    const defaultColumnIds = useMemo(() => columns.map((c) => c.id), [columns]);
-
-    const { timeline: { rowHeight, sampleSize } = timelineDefaults } = useSelector((state: State) =>
-      timelineBodySelector(state, timelineId)
-    );
+    const { timeline: { rowHeight, sampleSize, excludedRowRendererIds } = timelineDefaults } =
+      useSelector((state: State) => timelineBodySelector(state, timelineId));
```

Comment:
I don't think this selector works as it should, can you update it to use the new `selectTimelineById` that @PhilippeOberti added? Thanks!

Reply:
Done here :  886b161ef0e